src/coreclr/nativeaot/Runtime/eventpipe/ep-rt-aot.cpp

```diff
@@ -237,8 +243,6 @@ ep_rt_aot_atomic_inc_int64_t (volatile int64_t *value)
 {
     STATIC_CONTRACT_NOTHROW;
 
-    // shipping criteria: no EVENTPIPE-NATIVEAOT-TODO left in the codebase
-    // TODO: Consider replacing with a new PalInterlockedIncrement64 service
```

Comment:
Is there a reason we don't want to do this anymore?

Reply:
I think this is best done as part of performance analysis work, to find out the biggest areas of concerns and address those. We have #79241 and #86828 tracking performance.

Reply:
This should be part of https://github.com/dotnet/runtime/issues/87069 . It is cheaper to make good shared implementation than to try to figure out whether it has measurable performance impact.